test/test_sim_utils.py

```diff
@@ -222,3 +224,67 @@ def test_object_getters():
                     sim, link_object_id, ao_link_map
                 )
                 assert obj_from_id_getter.object_id == ao.object_id
+
+
+@pytest.mark.skipif(
+    not built_with_bullet,
+    reason="Raycasting API requires Bullet physics.",
+)
+@pytest.mark.skipif(
+    not osp.exists("data/replica_cad/"),
+    reason="Requires ReplicaCAD dataset.",
+)
+def test_keypoint_cast_prepositions():
+    sim_settings = default_sim_settings.copy()
+    sim_settings[
+        "scene_dataset_config_file"
+    ] = "data/replica_cad/replicaCAD.scene_dataset_config.json"
+    sim_settings["scene"] = "apt_0"
+    hab_cfg = make_cfg(sim_settings)
+    with Simulator(hab_cfg) as sim:
+        all_objects = get_all_object_ids(sim)
+
+        mixer_object = get_obj_from_handle(
+            sim, "frl_apartment_small_appliance_01_:0000"
```

Comment:
Are these sim entities stable enough to be included in tests, or should we be placing new entities?

Reply:
I'm assuming these are stable because they are ReplicaCAD scenes which we have no intention of modifying. This scene "apt_0" is a reconstruction of the first static "FRL apartment" arrangement from [Replica](https://github.com/facebookresearch/Replica-Dataset/tree/main). 

If anything were to change in the future it would most likely be dynamic settling which should preserve the tested relationships here.